src/System.Private.CoreLib/shared/System/Resources/ResourceManager.cs

```diff
@@ -122,11 +113,6 @@ internal class CultureNameResourceSetPair
         }
 
         protected string BaseNameField;
-        // Sets is a many-to-one table of CultureInfos mapped to ResourceSets.
-        // Don't synchronize ResourceSets - too fine-grained a lock to be effective
-        [Obsolete("call InternalGetResourceSet instead")]
-        internal Hashtable ResourceSets;
```

Comment:
This was not used anywhere and seems like a leftover from NetFX. It doesn't exist in CoreRT.